### ISSUE TITLE
Bot stays offline

### ISSUE THREAD

**Issue by facies** [discord-php/DiscordPHP#771]
Hello,

I try to run the basic example with php CLI on my webserver.
I get no errors but the bot stays offline.

Here is the output from the terminal:

[2022-03-29T17:46:12.940988+02:00] DiscordPHP.INFO: gateway retrieved and set {"gateway":"wss://gateway.discord.gg/?v=6&encoding=json","session":{"total":1000,"remaining":1000,"reset_after":0,"max_concurrency":1}} []
[2022-03-29T17:46:12.945512+02:00] DiscordPHP.INFO: starting connection to websocket {"gateway":"wss://gateway.discord.gg/?v=6&encoding=json"} []

Any ideas are welcome. Thank you for looking.

Best,

Facies

**Comment by key2peace** [comment]
update your discordphp, you are using a very old version

**Comment by facies** [comment]
Thank you very much for your help.

I updated php from 7.2.29 to 7.4.4

Then I try to update the package with the following command: composer update team-reflex/.discord-php
I have "team-reflex/discord-php": "^5.1" in composer.json

But it ouputs that:

Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating autoload files

This is the first time I'm using Composer, sorry for that and thanks a lot for your help.

Best,

Facies

**Comment by SQKo** [comment]
Yes, version 5.1 is very old, that's the maximum you can get if you were still using PHP 7.2

After upgrading to PHP 7.4, just remove the vendor folder (to clean up)
then do `composer require team-reflex/discord-php` again, that should point to the latest which is ^7.0

**Comment by facies** [comment]
Thanks a lot It worked and I was able to run the example with success.

You help is greatly appreciated, thanks again <3
